remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeStorageInfoRelation.java

```diff
@@ -72,4 +77,42 @@ public void afterPropertiesSet() throws Exception
     {
         PropertyCheck.mandatory(this, "storageInformation", storageInformation);
     }
+
+    @Experimental
+    @Operation("archive")
+    @WebApiDescription(title = "Request send content to archive",
+            description = "Submits a request to send content to archive",
+            successStatus = HttpServletResponse.SC_ACCEPTED)
+    public void requestArchiveContent(String entityResourceId, String id, Void voidRequest, Parameters parameters,
+                                      WithResponse withResponse)
+    {
+
+        final boolean result = storageInformation.requestArchiveContent(entityResourceId, id);
+        //TODO: handle below status codes and possible exceptions when downstream implementations are present
```

Comment:
TODO? Ticket reference? Is this needed? Only exceptions I can for see are `ArchivedIOException` or `IllegalArgumentException` These should not need additional handling.

Reply:
Thanks - removed that. Though I was thinking about `UnsupportedOperationException` versus a false return handling. 
Any other suggestion are welcomed.